README-dev.rst

```diff
@@ -126,6 +132,10 @@ and change ``tests/unit/`` to ``tests/``.
 
 Running the Benchmarks
 ======================
+There needs to be a version of cassandra running locally so before running the benchmarks, if ccm is installed:
+	
+	ccm create benchmark_cluster -v 3.0.1 && ccm populate -n 1 && ccm start
```

Comment:
This can be combined into a single ccm command:
`ccm create benchmark_cluster -v 3.0.1 -n 1 -s`

Was there a reason to chain commands?